ci-automation/packages-tag.sh

```diff
@@ -98,9 +98,13 @@ function _packages_tag_impl() {
           local ret=0
           git diff --exit-code "${existing_tag}" || ret=$?
           if [[ ret -eq 0 ]]; then
-            touch ./skip-build
-            echo "Creating ./skip-build flag file, indicating that the build must not to continue because no new tag got created as there are no changes since tag ${existing_tag}" >&2
-            return 0
+            if curl --head --fail --silent --show-error --location "https://${BUILDCACHE_SERVER}/images/amd64/${FLATCAR_VERSION}/flatcar_production_image.bin.bz2"
```

Comment:
```suggestion
            if curl --head --fail --silent --show-error --location "https://${BUILDCACHE_SERVER}/images/amd64/${FLATCAR_VERSION}/flatcar_production_image.bin.bz2" \
```

Otherwise it fails as currently:
```
17:20:48  jenkins/systemd-run-wrap.sh: ci-automation/packages-tag.sh: line 102: syntax error near unexpected token `&&'
17:20:48  jenkins/systemd-run-wrap.sh: ci-automation/packages-tag.sh: line 102: `              && curl --head --fail --silent --show-error --location "[https://${BUILDCACHE_SERVER}/images/arm64/${FLATCAR_VERSION}/flatcar_production_image.bin.bz2](https://${buildcache_server}/images/arm64/$%7BFLATCAR_VERSION%7D/flatcar_production_image.bin.bz2)"; then'
```

Reply:
Just seen the fixing commit. Thanks!

Reply:
Lesson learnt: run `shellcheck` all the time^^

Reply:
Might be we could add a simple github actions running shellcheck on PR?

Reply:
`shellcheck -x --severity=warning` is still a bit noisy until we fix the warnings, just `=error` isn't very helpful on the other hand.